features/readablestream-from.yml

```
@@ -0,0 +1,6 @@
+name: ReadableStream.from()
+description: The `ReadableStream.from()` static method wraps iterable and async iterable objects, such as arrays or async generator functions, as a readable stream.
```

"wraps … as" reads a little funny to me (though I see that MDN uses the same language). What do you think of this instead, which borrows a construction from the `iterator-methods` feature description?

```suggestion
description: The `ReadableStream.from()` static method converts an iterable or async iterable object, such as an array or async generator function, into a readable stream.
```